htdocs/fourn/commande/dispatch.php

```diff
@@ -258,7 +258,7 @@
 			}
 
 			// We ask to move a qty
-			if (GETPOST($qty) != 0) {
+			if (!empty(GETPOST($qty)) && GETPOST($qty) != 0) {
```

Comment:
It seems you can simplify into:
`if (GETPOST($qty)) {`
Does this simpler version fix your bug ?